packages/frontend/core/src/modules/collection/services/pinned-collection.ts

```
@@ -0,0 +1,71 @@
+
+import type {
+  PinnedCollectionRecord,
+  PinnedCollectionStore,
+} from '../stores/pinned-collection';
+
+export class PinnedCollectionService extends Service {
+  constructor(private readonly pinnedCollectionStore: PinnedCollectionStore) {
+    super();
+  }
+
+  pinnedCollections$ = LiveData.from<PinnedCollectionRecord[]>(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)